sphinx/src/orga/minutes/00_intro.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
Why? 🤔

Reply:
I was getting an error that it was not being referenced by any other page. Since I moved the orga content, minutes are left here and are handled in a separate PR